spinta/types/model.py

```diff
@@ -381,11 +384,30 @@ def check(context: Context, model: Model):
 
 @check.register(Context, Property)
 def check(context: Context, prop: Property):
+    commands.check(context, prop.dtype)
     if prop.enum:
         for value, item in prop.enum.items():
             commands.check(context, item, prop.dtype, item.prepare)
 
 
+@check.register(Context, DataType)
+def check(context: Context, dtype: DataType):
+    pass
+
+
+@check.register(Context, Ref)
+def check(context: Context, dtype: Ref):
+    if dtype.prop.level:
+        level = int(dtype.prop.level)
+        ref_external = dtype.model.external
+        external = dtype.prop.model.external
+        if ref_external and external and \
+                ref_external.resource and \
+                ref_external.resource != external.resource and \
+                level > 3:
```

Comment:
Now I realized, that checking external resource is a bad idea. Instead we should check that ref property level is not higher than referenced models pkeys. Basically we should check `zip(dtype.refprops, dtype.model.external.pkeys)` and each a.level <= b.level.